conmon-rs/server/src/rpc.rs

```diff
@@ -61,6 +63,10 @@ impl conmon::Server for Server {
         let bundle_path = PathBuf::from(pry!(req.get_bundle_path()));
         let pidfile = bundle_path.join("pidfile");
         debug!("PID file is {}", pidfile.display());
+
+        let attach_path = bundle_path.join("attach");
+        self.attach = pry_err!(Attach::new(&attach_path).context("create attach endpoint")).into();
```

Comment:
we need an attach socket per container, can we store this in the child struct?

Reply:
(we'll eventually need an attach socket per Exec (not ExecSync) request as well)

Reply:
Fixed, moved into the reapable child.